raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala

```diff
@@ -42,14 +43,14 @@ object Rasterizer {
           array(row * cols + col) = value
     foreachCellByGeometry(geom, rasterExtent)(f2)
     ArrayTile(array, rasterExtent.cols, rasterExtent.rows)
-  } 
+  }
 
   /**
    * Create a raster from a geometry feature.
    * @param feature       Feature to rasterize
    * @param rasterExtent  Definition of raster to create
    * @param f             Function that takes col, row, feature and returns value to burn
-   */ 
+   */
   def rasterize(feature: Geometry, rasterExtent: RasterExtent)(f: Transformer[Int]) = {
```

Comment:
We don't need a Transformer, it should just be a function `(Int, Int) => Int`. This is an artifact of the old code using a Function3, which is not specialized, so using a type avoided boxing.